dev/envvars.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+case "$(uname -s | tr '[:upper:]' '[:lower:]')" in
+  linux*)
+    if [ "$(ldd /bin/ls | grep -m1 'musl')" ]; then
```

Comment:
I haven't run this command myself but if you say this works I'll take your word for it 😄

Reply:
This is a copy-paste code from [instrument.sh](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/c7b279bcdf3510e17e6d32707b5ba919f60488ed/instrument.sh#L10) which works as expected. I think @pellared wrote the logic for it and tested out.